### ISSUE TITLE
Request

### ISSUE THREAD

**Issue by webdevnerdstuff** [ModyQyW/vite-plugin-stylelint#60]
### Describe the problem

I can't remember exactly what causes this (I believe the github release), but when installing a fresh copy of `vite-plugin-stylelint` it is installing the beta version, which is not fully working yet (thus beta). It also updates to the latest beta when updating packages to the latest releases.  I'm sure this will cause issues for people when first installing it, as it's causing nothing to be linted or fixed unfortunately. A few times lately when I have been updating packages I got caught with this problem, and each time I kept forgetting the beta got installed instead of the last working version.

To fix this I think you would need to create a release for [v5.3.1](https://github.com/ModyQyW/vite-plugin-stylelint/releases/tag/v5.3.1) and set it as the "Latest Release". I think this should avoid the beta being installed on users.

### Reproduction

```bash
pnpm up -i -L
```
or
```
pnpm add -D vite-plugin-stylelint
```

### System Information

```Shell
n/a
```

### Package manager used

pnpm

### Verification
   
- [X] Follow our [Code of Conduct](https://github.com/uni-helper/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read [the contribution guidelines](https://github.com/antfu/contribute) and [notes from a tired maintainer](https://github.com/ModyQyW/tired-maintainer) .
- [X] Check if there is already an issue reporting the same error to avoid creating a duplicate.
- [ ] This is a specific bug. Please open a GitHub discussion to discuss your issue.
- [X] The provided reproduction is [a minimal reproduction](https://stackoverflow.com/help/minimal-reproducible-example) of the problem .

### contribute

- [ ] I'd like to submit a PR to fix this issue
- [ ] I'd like to submit a PR with failing tests (just do it, thanks!)

**Comment by ModyQyW** [comment]
Can you tell me what the specific problem is with the beta version?

**Comment by webdevnerdstuff** [comment]
When running vite it doesn't seem to output anything when I purposely add things to test breaking it and cause an error. Works fine with v5.3.1.

I threw in an error on a repo I'm currently working on to give you an example.

https://github.com/webdevnerdstuff/vuetify-stepper-form/tree/stylelint-ex

in the file `src/documentation/DocsPage.vue` I added
```css
.foo {
  color: red;
}
```

This should throw an error based off the rules I have. You can see that set here:
https://github.com/webdevnerdstuff/stylelint-config-wdns/blob/main/index.js#L88

It doesn't matter that I'm extends those rules from the package I created, as the same problem occurred before I made a package to extend.

But the main issue I'm asking in the request is that you set v5.3.1 as the latest release, so when users add the package they are not getting the beta version until it's the final version and latest.

**Comment by ModyQyW** [comment]
> But the main issue I'm asking in the request is that you set v5.3.1 as the latest release, so when users add the package they are not getting the beta version until it's the final version and latest.

Yes, I understand what you asked for. I think v6 beta is stable enough and a beta version should be good. I am trying to fix this issue and release v6 stable then. Also, I have made v5.3.1 the latest version for now. Thanks for your feedback! 🙏

**Comment by webdevnerdstuff** [comment]
If you need anything else from me just let me know. I have that branch separate that you can fork/mess with etc if you need it.

**Comment by ModyQyW** [comment]
> If you need anything else from me just let me know. I have that branch separate that you can fork/mess with etc if you need it.

Extremely thanks!

Just released v6.0.0-beta.2 as the next tag which may be the last version before v6.0.0.

v5.3.1 still is the latest version for now.
